docassemble/CLAGuardianship/data/questions/petition_for_removal_of_guardian.yml

```diff
@@ -667,9 +822,13 @@ sets:
   - responsible_parents[i].address.zip
   - responsible_parents[i].address.country
   - responsible_parents[i].phone_number
-  - responsible_parents[i].appointment_date
 question: |
-  ${ capitalize(ordinal(i)) } parent resuming responsibility for
+  % if responsible_parents.target_number > 1:
+  ${ capitalize(ordinal(i)) } parent
+  % else:
+  Parent
+  % endif 
+  resuming responsibility for
   % if who_is_making_petition == "minor" and (not form_filled_by_attorney):
   you
   % else:
```

Comment:
Avoid putting conditional logic into the middle of a sentence - it makes fragments that make it hard to translate into languages with different common word orders, like Spanish.